spec/models/manifest_uploader_spec.rb

```diff
@@ -61,4 +65,103 @@
       expect(labwares).to be_empty
     end
   end
+
```

Comment:
Is it worth adding a test where the manifest has a mixture of ordered and unordered locations?

Reply:
I've now added this 👍